src/main/java/pl/zimi/zadanka/Zad3.java

```diff
@@ -0,0 +1,24 @@
+package pl.zimi.zadanka;
+
+public class Zad3 {
+    public boolean checkPowerOfTen(Integer inputNumber) {
```

Comment:
isPowerOfTen

ta metoda odpowiada nam na pytanie "czy", w takich wzorcach używamy "is" jako przedrostek
to dobre wzorce nazewnicze